scripts/microbe_stage/population_system.lua

```diff
@@ -0,0 +1,93 @@
+
+POPULATION_SIMULATION_INTERVAL = 1200
+
+--------------------------------------------------------------------------------
+-- Population
+--
+-- Holds information about a specific population (species \intersect patch)
+--------------------------------------------------------------------------------
+class 'Population'
+
+function Population:__init(species)
+    self.species = species
+    self.heldCompounds = {} -- compounds that are available for intracellular processes
+    self.lockedCompounds = {} -- compounds that aren't, but will be released on deaths
+end
+
+--[[
+There are probably some functions here that would be useful to package with the population,
+possibly useful enough to make this a component and have a system that coordinates.
+
```

Comment:
yeah I'm sorry :p
You simply started on a simplified version of the population stuff right when we'd cracked the secret to making the complicated full simulation work, so instead of adding then removing your code I decided I should just build the species model and lay groundwork for the rest